### PR TITLE
Sync kafka settings

### PR DESCRIPTION
**Pull request by hackaugusto**
- update kafka/zookeeper container versions
- change from cp-server to cp-kafka which is the community edition
- made sure the docker-compose and the pytest fixtures use the same settings

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -23,8 +23,10 @@
 import socket
 import time
 
+# Keep these in sync with the Makefile
 KAFKA_CURRENT_VERSION = "2.4"
-BASEDIR = "kafka_2.12-2.4.1"
+BASEDIR = "kafka_2.13-2.7.0"
```
Comment (tvainika) [inline tests/integration/conftest.py:28]: The line above has `KAFKA_CURRENT_VERSION = "2.4"`, but this PR switches to Kafka 2.7. I see that KAFKA_CURRENT_VERSION is used only for protocol version related settings, but reflecting to comment line before is confusing. What software versions and protocol versions should be defined and where?

**Review by tvainika — CHANGES_REQUESTED** [review]
Version sync comments and version usage needs a bit clarification

File: tests/integration/conftest.py (hunk 1 above)
Comment (hackaugusto) [reply tests/integration/conftest.py:28]: I'm not sure why `KAFKA_CURRENT_VERSION` is necessary, I just synchronized the files. Unfortunately setting it to `2.7.0` crashes Kafka

Comment (tvainika) [reply tests/integration/conftest.py:28]: I'm fine with updating the comments here (and possibly in Makefile) about what versions need to be in sync. Also what about renaming  `KAFKA_CURRENT_VERSION` to something like `KAFKA_PROTOCOL_VERSION`?

Comment (hackaugusto) [reply tests/integration/conftest.py:33]: I must have used an outdated version of the container image. I did check the documentation and tried it again, and the protocol version set as the current version works.

**Review by tvainika — APPROVED** [review]

**Pull request merged**